app/models/concerns/to_csv.rb

```diff
@@ -8,12 +8,25 @@ def dashboard
       all
     end
 
+    # def column_names
+
+    # end
+
     # @return [String]
     def to_csv
+      formatted = CoercionDecorator.new(dashboard).call
```

Comment:
We thought at the outset we'd want to initialise with the input but we could pass it to the method. Pros and cons of a single instance, we don't leverage type checking from initialiser so we don't lose that. What do you think? It'd move out and get memoised.